crates/sp-domains-fraud-proof/src/lib.rs

```diff
@@ -387,6 +389,7 @@ pub enum StatelessDomainRuntimeCall {
     },
     IsInherentExtrinsic(OpaqueExtrinsic),
     IsDecodableExtrinsic(OpaqueExtrinsic),
+    IsValidDomainSudoCall(Vec<u8>),
```

Comment:
This updates the argument of the host function, I'm not sure if it is compatible since the runtime is expecting the host function without the new variant while the client is providing a host function that included the new variant.

Reply:
This enum variant is added in the last. So enum index should not be an issue since Current runtime would not be aware of this IMHO.
Thoughts @nazar-pc ?